### PR TITLE
Update Japanese translation

### PR DESCRIPTION
**Pull request by tkusano**
@Hima-Zinn  @dolciss @middlingphys @hibiki , review please

- remove obsolete messages for font related nux
- translate some error messages

### CODE REVIEW

File: src/locale/locales/ja/messages.po [hunk 1]
```diff
@@ -6945,7 +6929,7 @@ msgstr "リストでのミュートを解除"
 
 #: src/lib/strings/errors.ts:11
 msgid "Unable to connect. Please check your internet connection and try again."
-msgstr ""
+msgstr "接続できません。インターネット接続をチェックして再度試してください。"
```
Comment (Hima-Zinn) [inline src/locale/locales/ja/messages.po:6932]:
```suggestion
msgstr "接続できません。インターネット接続を確認して再度試してください。"
```
1273行辺りの表現と一致させるため。

**Review by Hima-Zinn — COMMENTED** [review]
Please check my review.

**Comment by tkusano** [comment]
memo:
https://bsky.app/profile/l-tan.dolciss.net/post/3lbkarxove22v

**Comment by pfrazee** [comment]
We good to go?

**Comment by Hima-Zinn** [comment]
Sorry, we will have to wait until tkusano's responds.

File: src/locale/locales/ja/messages.po (hunk 1 above)
Comment (tkusano) [reply src/locale/locales/ja/messages.po]: fixed

**Comment by tkusano** [comment]
ラベルの訳語について:
https://bsky.app/profile/pomcan.bsky.social/post/3lbohhg5hmc26

**Comment by tkusano** [comment]
translate new messages:
- https://github.com/bluesky-social/social-app/commit/5d4aaa5b5faa595d1967eabe11fda9615d7ba1f6
- https://github.com/bluesky-social/social-app/commit/ac5b2cf31f2bb45f1bf8a180705249d3cce8017d
- https://github.com/bluesky-social/social-app/commit/631277dc3d8d666f09e86bd390c90239738853c9
- https://github.com/bluesky-social/social-app/commit/378107492194a5f408747790015c4ca1d624302b

File: src/locale/locales/ja/messages.po [hunk 2]
```diff
@@ -5702,17 +5690,9 @@ msgstr "Blueskyの求人を見る"
...
-#~ msgstr "シークバー"
-
 #: src/view/com/util/post-embeds/VideoEmbedInner/web-controls/Scrubber.tsx:190
 msgid "Seek slider. Use the arrow keys to seek forwards and backwards, and space to play/pause"
-msgstr ""
-
-#: src/view/com/util/Selector.tsx:107
-#~ msgid "Select {item}"
-#~ msgstr "{item}を選択"
+msgstr "シークバー。矢印キーで使って早送りや早戻し、スペースで再生／停止"
```
Comment (middlingphys) [inline src/locale/locales/ja/messages.po:5695]:
```suggestion
msgstr "シークバー。矢印キーで早送りや早戻し、スペースで再生／一時停止"
```

Comment (tkusano) [reply src/locale/locales/ja/messages.po]: applied

**Comment by vyv03354** [comment]
https://github.com/bluesky-social/social-app/blob/6a2067a761aa99523b83cf938f3272c680856914/src/locale/locales/ja/messages.po#L6439
1.96での変更点ではないのですが、これは右ナビゲーションに表示される利用規約へのリンクに使われるので、「規約」と訳すべきではないでしょうか。

**Comment by tkusano** [comment]
translate new messages and remove obsolete entries.

translation note:
- 'drawer' is already translated as '引き出し', so I translated 'drawer menu' as '引き出しメニュー'

**Comment by tkusano** [comment]
translate changed feed copy https://github.com/bluesky-social/social-app/commit/5297973f682d31b1f0cd312a4fa2ab42c8103c1e

**Comment by tkusano** [comment]
update the translation changed by https://github.com/bluesky-social/social-app/commit/e052f5e198603246cb031e00d9cadc2ae4bb140d

**Comment by tkusano** [comment]
- translate count messages on repost/quote/likedby screen
- app icon related messages

plural messages like "# quotes" are usually translated as "#引用" by removing the space between the number and the phrase, but I left the space intentionally to ensure consistency with existing screens.

File: src/locale/locales/ja/messages.po [hunk 3]
```diff
@@ -46,10 +58,18 @@ msgstr "{0, plural, other {#分}}"
...
+#: src/screens/Post/PostQuotes.tsx:41
+msgid "{0, plural, one {# quote} other {# quotes}}"
+msgstr "{0, plural, other {# 引用}}"
+
 #: src/view/com/util/post-ctrls/RepostButton.tsx:69
 msgid "{0, plural, one {# repost} other {# reposts}}"
 msgstr "{0, plural, other {#回のリポスト}}"
 
+#: src/screens/Post/PostRepostedBy.tsx:41
+msgid "{0, plural, one {# reposts} other {# reposts}}"
```
Comment (surfdude29) [inline src/locale/locales/ja/messages.po:70]: FYI #7064 just got merged to fix a typo here (thanks @auroursa!)

Comment (tkusano) [reply src/locale/locales/ja/messages.po]: thanks for your notification!!

**Comment by auroursa** [comment]
#7065 made some adjustments to plural words, maybe it needs to be re-translated based on this.

**Comment by tkusano** [comment]
track #7065

**Comment by tkusano** [comment]
translated ProfileFeed related messages https://github.com/bluesky-social/social-app/commit/2808f8b73da4791e0f3cb85b03403738e5bf6b63

**Comment by tkusano** [comment]
- follow new ProfileLIst design https://github.com/bluesky-social/social-app/commit/da6bcc54cfff198e629a34987746028414867ec9
- translate change handle text https://github.com/bluesky-social/social-app/commit/ca8ca903b271e9f7c901b61cad7580ed7a2b756e
image

**Comment by tkusano** [comment]
track https://github.com/bluesky-social/social-app/commit/1ac307bc42bdf370c2011e7e7f2c76baea3441d2

**Comment by auroursa** [comment]
#7136 updated the translations to prevent fallback after release, but this may cause newly edited translation strings to not be flagged as missing. I believe we should adjust this. Here are my suggestions:

`#: src/screens/Signup/StepHandle.tsx:114`
`Type your desired username`
`あなたのユーザー名を入力`

`#: src/screens/Signup/StepHandle.tsx:125`
`Your full username will be`
`フルユーザー名は`

`#: src/screens/Signup/index.tsx:142`
`Choose your username`
`ユーザー名をお選びください`

Additionally, I noticed that the translation at `#: src/screens/Signup/state.ts:210` is `パスワードを選択してください`. Perhaps we could standardize this and adopt a consistent translation for `Choose`?

ご考慮いただき、ありがとうございます

**Comment by tkusano** [comment]
> #7136 updated the translations to prevent fallback after release, but this may cause newly edited translation strings to not be flagged as missing. I believe we should adjust this. Here are my suggestions:
> 
> `#: src/screens/Signup/StepHandle.tsx:114` `Type your desired username` `ご希望のユーザー名を入力してください`
> 
> `#: src/screens/Signup/StepHandle.tsx:125` `Your full username will be` `フルユーザー名は`
> 
> `#: src/screens/Signup/index.tsx:142` `Choose your username` `ユーザー名をお選びください`
> 
> Additionally, I noticed that the translation at `#: src/screens/Signup/state.ts:210` is `パスワードを選択してください`. Perhaps we could standardize this and adopt a consistent translation for `Choose`?
> 
> ご考慮いただき、ありがとうございます

updated. thanks.

bluesky-signup-screen1
bluesky-signup-screen2

**Comment by tkusano** [comment]
translated topic related UI

image
image
image
image
image

**Review by pfrazee — APPROVED** [review]
Awesome, thank you everybody!

**Pull request merged**